client.go

```diff
@@ -51,6 +59,7 @@ type Client struct {
 	successCount int64
 	errorCount   int64
 	debug        bool
+	Insecure     bool
```

Comment:
Can we change Insecure to insecure. (For consistency, and also capitalization in go differentiates private/public)